README.md

```diff
@@ -113,7 +112,7 @@ A variable template that always produces IRIs.
     {
       "@type": "VariableTemplateNamedNode",
       "name": "person",
-      "substitutionProvider": { ... }
```

Comment:
Can we keep these? It's to indicate that content is needed here.

Reply:
It breaks the linter, though, but I can see if it is possible to add an eslint-ignore statement in a Markdown file as a comment (never tried it).

Reply:
I disabled linting for the readme and reverted these changes, I think that is the best solution for now, since it breaks the actual parsing itself for eslint.

Reply:
Or, we can mark it as `json5`, and replace the `...` with a comment (which is supported in JSON5).

Reply:
@rubensworks It seems `json` is enough for it to accept comments, as well. This should now be fixed.